### ISSUE TITLE
How to compile master branch?

### ISSUE THREAD

**Issue by liyishuai** [PrincetonUniversity/VST#427]
On Coq 8.11.1, `make` fails with:
```
File "./compcert/x86_32/Archi.v", line 21, characters 15-21:
Error: Cannot find a physical path bound to logical path matching suffix
<> and prefix Flocq.
```

**Comment by andrew-appel** [comment]
TL;DR:  Fix this by commenting out the line   `From Flocq`   in the four files found by `cd compcert; grep 'From Flocq' */*.v`

Explanation: Right now we are in transition from "CompCert-with-internal-Flocq" to "CompCert-with-external-Flocq".  That is, until now, standard CompCert has its own internal copy of the Flocq floating-point library, but in future releases we want to connect to other floating-point reasoning tools that are based on the standard user-contrib (or opam) version of Flocq.  So you have two choices:
1.  Build the master branch in a Coq that has access to Flocq (via opam, or via a "package");  or,
2.  Revert to using CompCert-with-internal-Flocq by doing the fix described in the first line of this message.

**Comment by andrew-appel** [comment]
@liyishuai, if this adequately addresses your problem, please close this issue report.
